cmd/security-agent/app/app.go

```diff
@@ -116,13 +116,13 @@ func init() {
 	SecurityAgentCmd.AddCommand(startCmd)
 }
 
-func newLogContext(logsConfig *config.LogsConfigKeys, intakeTrackType, endpointPrefix string) (*config.Endpoints, *client.DestinationsContext, error) {
-	endpoints, err := config.BuildHTTPEndpointsWithConfig(logsConfig, endpointPrefix, intakeTrackType, config.DefaultIntakeProtocol)
+func newLogContext(logsConfig *config.LogsConfigKeys, intakeTrackType, intakeSource, endpointPrefix string) (*config.Endpoints, *client.DestinationsContext, error) {
```

Comment:
I think it might be better to order parameters in the same way they are ordered in other functions for consistency.

```suggestion
func newLogContext(logsConfig *config.LogsConfigKeys, endpointPrefix, intakeTrackType, intakeSource string) (*config.Endpoints, *client.DestinationsContext, error) {
```